source/mainnet/smart-contracts/onboarding-guide-ethereum-developers/faq.rst

```diff
@@ -0,0 +1,371 @@
+.. _faq:
+
+====
+FAQs
+====
+
+This is a list of Frequently Asked Questions about Concordium. It is focused on
+helping developers with an Ethereum/solidity background to understand
+the Concordium blockchain and its smart contract ecosystem.
+
+Feel free to suggest additional FAQs by contacting `support@concordium.software`.
+
+.. dropdown::  Example question?
+
+    Example text
+
+    Example picture
+
+    .. image:: ./images/onboarding_ethereum_developers_1.png
+      :width: 100 %
+
+    Example code
+
+    .. code-block:: console
+
+        $./concordium-client contract init piggy_bank_part2_module --sender <account-name> --contract PiggyBank --name piggy_bank_part2_instance --energy 1000 --grpc-port 10001
+
+    Example note
+
+    .. note::
+
+        abcdefgh
+
+Concordium smart contracts:
+===========================
+
+.. dropdown::  What smart contract language is used on Concordium?
+
+    Concordium uses Rust as smart contract language.
```

Comment:
I am not sure what the ambition here is, but do we perhaps want to be more precise. Like on ethereum, we have two levels. The Rust is what the user typically writes. But on chain we only have Wasm.

So it is more correct that Wasm is what "Concordium uses" and Rust is the language for which we have best support.